src/deluge/gui/views/view.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "gui/views/view.h"
 #include "definitions_cxx.hpp"
+#include "deluge/model/settings/runtime_feature_settings.h"
```

Comment:
Actually you can immediately put the non-song level stutter quantization setting in the sound menu

Reply:
If i put the stutter quantize option inside the sound menu of each clip, how then can you change the quantize settings for the global song mode?

Reply:
You can't, it would use the value from the defaults menu

Reply:
Would it then make sense to include options for quantize (on/off) and quantize sync type (even/triplet/dotted)? The sync level will be controlled by the golden knob turning and the values instead of going in the full range  0-127 which in the extremes produces ridiculous and useless buffer sizes, use the more musical values equivalent to 4ths through 64ths (in the case of even syncs the values are 48, 56, 64, 72, 80, corresponding to 4th, 8th, 16th, 32th, 64th). I don't know if there would be some stutter values that produce triplets or dotted buffer sizes, i need to test

Reply:
Ya that'd make sense to me

Reply:
I have tested each stutter rate and it was impossible to find a perfect triplets or dotted note. So I assume the only possible stutter values that we can quantize to musical rates are the even values. That is, no need for tripplets or dotted settings as it is impossible with the current stutter implementation to get those working.

I am not so sure also about moving the code to each individual sound and having then a separate stutter quantization setting for affect-all kits and global song in the Defaults menu. Too cluttered. if you are going to perform and use the stutter effect I think you would be used to having one global setting for the whole song and perform with that in mind. In my case for example I would probably leave it quantized forever as it becomes more useful to me to help me select a rate that is a musical division for the initial buffer recording and then play with that buffer it the normal way.